app/views/homepage/_government_activity.erb

```diff
@@ -0,0 +1,75 @@
+<!--
```

Comment:
A tiny little nitpick that this should be `government_activity.html.erb` not just `.erb`